github/feeds.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+// FeedsService provides access to the feeds related functions
+// in the GitHub API.
+//
+// GitHub API docs: https://developer.github.com/v3/activity/feeds/
+type FeedsService struct {
```

Comment:
This is part of the "Activity" portion of the GitHub docs, so this would go on the existing `ActivityService`, rather than creating a new feeds service.